test/Microsoft.AspNet.Security.Test/project.json

```diff
@@ -18,7 +20,8 @@
     "frameworks": {
         "aspnet50": {
             "dependencies": {
-                "Shouldly": "1.1.1.1"
+                "Shouldly": "1.1.1.1",
+                "System.Security.Claims": "1.0.0-*"
```

Comment:
Shouldly is not available for "aspnetcore50". What guidance does the K team have here?